ledger/internal/prefetcher/prefetcher.go

```diff
@@ -402,7 +441,8 @@ func (p *accountPrefetcher) prefetch(ctx context.Context) {
 			// if we had no error, write the result to the output channel.
 			// this write will not block since we preallocated enough space on the channel.
 			p.outChan <- LoadedTransactionGroup{
-				TxnGroup:  p.groups[next],
+				Err:       groupsReady[next].err,
```

Comment:
The above comment needs an update since we now queue the group regardless of error, right?

Reply:
Done